terraform/gcp/variables.tf

```diff
@@ -0,0 +1,158 @@
+variable "PROJECT_ID" {
```

Comment:
Same here, wondering if we could have one `variables.tf` file for all cloud providers, and maybe adapt it at startup (e.g. on `--init` command)

Reply:
We can not use same variable file for both AWS and GCP. We have created 3 files. aws_vars, gcp_vars and common_vars. I hope this solves the issue.